ngsderive/utils.py

```diff
@@ -415,9 +409,6 @@ def query(self, chr, start, end):
             hits.append(result)
         return hits
 
-    def next(self):
```

Comment:
Were these dropped basically because you didn't want to support Python 2 for these classes? Of course, I think that's reasonable, just checking to make sure my understanding is correct.

Reply:
Yes. These methods aren't needed in Python3, which is all we support, so they were just extra lines of code not doing anything. You were the one that pointed we don't need them anymore in your first review.